### ISSUE TITLE
Dynamically set the last updated date

### ISSUE THREAD

**Issue by aciccarello** [aciccarello/aciccarello.github.io#17]
Ideally, we could do this from an HTTP header but I'm not aware of any accurate header. Also may want to have different date per page.

### Alternative:
Use [Github API](https://developer.github.com/v3/repos/commits/)

```
GET https://api.github.com/repos/aciccarello/aciccarello.github.io/commits
```

Only populate value if request is successful to avoid an empty "Last update:" in footer.

**Comment by aciccarello** [comment]
Need to check the accuracy of the `last-modified` header.

**Comment by aciccarello** [comment]
Closed via: 6cbc7a3
